code/modules/clothing/clothing.dm

```diff
@@ -331,7 +331,7 @@ BLIND     // can't see anything
 /obj/item/clothing/under/New()
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
-		sensor_mode = pickweight(list(0 = 1, 1 = 2, 2 = 3, 3 = 2))
+		sensor_mode = pick(0, 1, 1, 2, 2, 2, 3, 3))

```

Comment:
it would probably help with readability.

Also I think this actually sets the weights differently to what you want? Looks like the highest level has more chance than the second highest, just from reading this.